components/WelcomeSection.vue

```diff
@@ -1,21 +1,10 @@
 <template>
     <div class="flex flex-col bg-gray-800 p-12 w-full">
-        <span class="text-4xl text-gray-100"> The open-source project that's helping Japan get vaccinated </span>
-        <div class="mt-8 aspect-w-10 aspect-h-6 sm:aspect-w-2 sm:aspect-h-1 lg:aspect-w-10">
-            <img class="w-3/4 mb-8 rounded-2xl shadow-2xl" src="../assets/images/homepage/findadoc-boostershot-wiki.png"
-                alt="" />
-        </div>
-        <span class="text-lg text-gray-300">
-            Members of the international community are working together to collect and share booster shot information in
-            Japan. We've created a wiki that anyone in the community can edit and
-            contribute to.✨
-        </span>
-        <a href="https://info.findadoc.jp">
-            <button type="button"
-                class="my-8 inline-flex items-center px-6 py-3 border border-transparent text-base font-medium
-                    rounded-md shadow-sm text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500">
-                Get Booster Shot Info
-            </button>
-        </a>
+        <h1 class="text-gray-100&quot;">
```

Comment:
For SEO and a11y it's better to use `<h1>` and `<h2>`, etc., tags rather than adjusting the fonts through CSS, which has no semantic meaning.